ansible/roles/distributed_press/tasks/main.yml

```diff
@@ -41,16 +41,25 @@
     cd {{distributed_press_source}}
     npm ci
 
-- name: Nuke cache and protocol config
-  become: yes
-  become_user: "{{distributed_press_user}}"
-  shell: |
-    cd {{distributed_press_source}}
-    npm run nuke
+#- name: Nuke cache and protocol config
```

Comment:
isn't this needed in case a protocol change is made during an upgrade? or should this be done manually (does ansible have a way of conditionally running steps)

Reply:
Yeah I think we should do this manually. Or in the future maybe we can also detect when an upgrade is needed.